### DISCUSSION TITLE
How to upgrade from 11.3.1 to 2024.2

### DISCUSSION THREAD

**Discussion by baladpi** [pimcore/pimcore#17378] [Q&A / Support] [answered]
any document is available for this please share / step to do it

Also, I like to know which version is the latest 11.3.1 or 2024.2 - because I didn't see any release note for 2024.2

Please help me in this for understanding - Thanks

**Comment by baladpi** [comment]
is both are same?

**Reply by kingjia90** [reply, marked as answer]
`2024.2` is the platform version, check it out here https://github.com/pimcore/platform-version/tree/2024.2
It would install 1[1.3.1 or any further version ](https://github.com/pimcore/platform-version/blob/cac3ed6dcdb94893783e4835a453f4b8f2df97f3/composer.json#L45)that is supported by it

More explanation and info at 
https://pimcore.com/docs/platform/Platform_Version/
https://github.com/pimcore/skeleton?tab=readme-ov-file#pimcore-platform-version
